src/Tizen.Multimedia.Vision/MediaVision/InferencePoseLandmarkDetectorResult.cs

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2024 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.Collections.Generic;
+using InteropPLD = Interop.MediaVision.InferencePoseLandmarkDetection;
+
+
+namespace Tizen.Multimedia.Vision
+{
+    /// <summary>
+    /// Represents the result of <see cref="InferencePoseLandmarkDetector"/> operations.
+    /// </summary>
+    /// <since_tizen> 12 </since_tizen>
+    public class InferencePoseLandmarkDetectorResult
+    {
+        internal InferencePoseLandmarkDetectorResult(IntPtr handle)
+        {
+            InteropPLD.GetResultCount(handle, out ulong requestOrder, out uint count).
+                Validate("Failed to get result count.");
+
+            RequestOrder = requestOrder;
+            var positions = new List<Point>();
+
+            for (uint i = 0 ; i < count ; i++)
+            {
+                InteropPLD.GetPosition(handle, i, out uint x, out uint y).Validate("Failed to get position.");
+                positions.Add(new Point((int)x, (int)y));
+            }
+
+            Position = positions;
+        }
+
+        /// <summary>
+        /// Gets the requested order.
+        /// </summary>
+        /// <value>The requested order.</value>
+        /// <since_tizen> 12 </since_tizen>
+        public ulong RequestOrder { get; }
+
+        /// <summary>
+        /// Gets the position of the detected pose landmark.
+        /// </summary>
+        /// <since_tizen> 12 </since_tizen>
+        public IEnumerable<Point> Position { get; }
```

Comment:
having Position in the context of Post Landmark might be a little confusing, it's not a big deal but you might consider name like "Points", "Landmarks", "Vertexes", or ignore that as I admin it is a little nitpick

Reply:
I'll change it to `Points`